packages/valory/skills/decision_maker_abci/behaviours/base.py

```diff
@@ -56,6 +57,7 @@
 # which is what we want in most cases
 # more info here: https://safe-docs.dev.gnosisdev.com/safe/docs/contracts_tx_execution/
 SAFE_GAS = 0
+CID_PREFIX = "f01701220"
```

Comment:
afaik you can import this from autonomy somewhere as well

Reply:
Checked it out. It is in `autonomy.chain.metadata` and would require adding `open-autonomy` as a dependency to the skill :thinking: